strict-equal.js

```diff
@@ -0,0 +1,6 @@
+function strictEqual(sampleData) {
```

Comment:
Your definition for this function is unique.  Others set it up with two parameters such as `function strictEqual(itemOne, itemTwo) {`